conans/client/api/helpers/search.py

```diff
@@ -0,0 +1,20 @@
+from collections import OrderedDict
+
+from conans.search.search import search_recipes
+
+
+class Search(object):
```

Comment:
This class is maybe a bit boilerplate at this stage, 2 bare functions would make it better.